ansible_collections/arista/avd/docs/input-variable-validation-BETA.md

```diff
@@ -15,18 +15,22 @@ For details on the schema format see the [Schema Details](#schema-details) secti
 
 ## Automatic Type Conversion
 
-The validation of input variables is performed by the `arista.avd.validate` Action Plugin. The plugin also supports automatic
-conversion of variable types, if configured in the schema. The supported types that can be converted from depends on the type to
-be converted to. The current implementation supports the following automatic conversions:
+The validation of input variables is performed in most AVD Action Plugins. The validation also supports automatic conversion of
+variable types, if configured in the schema. The supported types that can be converted from depends on the type to be converted to.
+The current implementation supports the automatic conversions listed below. Conversions will only happen if set in the schema.
```

Comment:
This `Conversions will only happen if set in the schema` is redundant with this `The validation also supports automatic conversion of
variable types, if configured in the schema.`

Reply:
Yeah I will clean it up